hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -496,6 +496,12 @@ private FlinkOptions() {
   //  Compaction Options
   // ------------------------------------------------------------------------
 
+  public static final ConfigOption<Boolean> COMPACTION_BATCH_MODE_ENABLED = ConfigOptions
+          .key("compaction.batch.mode.enabled")
+          .booleanType()
+          .defaultValue(false) // default false for MOR write
```

Comment:
Can we remove this option ?

Reply:
👌🏻

Reply:
But I need a parameter to tell `CompactFunction` which to use `asyncCompaction` or `syncCompaction`, Batch mode must use `syncCompaction`.

Reply:
You can overwrite the option in hoodie table sink, when we know that the source is bounded.

Reply:
Our thoughts are the same.